### ISSUE TITLE
Accessibility issues with your choice of colours

### ISSUE THREAD

**Issue by maxgerber** [AbdullahChaudhry/week-5-CADJ#33]
The dev tools have flagged up a problem with the contrast ratio y'all have. The background picture is pretty nice! But it might mess with partially sighted people. As a suggestion, you could have a semi transparent black or grey background for the text using the rgba property in CSS 
  ```background:rgba(255,255,255,0.5)```, kind of like
![this?](https://divibooster-6d4b.kxcdn.com/wp-content/uploads/2014/10/divi-theme-semi-transparent-background.png)
